shoes-swt/lib/shoes/swt/text_block.rb

```diff
@@ -44,10 +44,11 @@ def contents_alignment(current_position)
       end
 
       def adjust_current_position(current_position)
-        current_position.y = @dsl.absolute_bottom
+        current_position.y = @dsl.absolute_bottom + 1
 
         last_segment = segments.last
         if last_segment && !@bumped_to_next_line
+          current_position.x -= 1
```

Comment:
This looks like it might be hiding another problem, I'm not sure - might be worth to put a comment or something there :)